test/SlowTests/Server/Documents/PeriodicBackup/Retention.cs

```diff
@@ -238,5 +276,19 @@ private static S3Settings GetS3Settings(string databaseName, [CallerMemberName]
                 RemoteFolderName = $"{remoteFolderName}/{databaseName}"
             };
         }
+
+        private const string AzureAccountName = "devstoreaccount1";
```

Comment:
we already have it in a different test. Let's unify this.

Reply:
done